go.mod

```diff
@@ -101,3 +101,7 @@ require (
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 	sigs.k8s.io/yaml v1.2.0 // indirect
 )
+
+replace (
```

Comment:
Maybe it should be updated before merging this PR.

Reply:
Here is the corresponding proto pull request, PTAL https://github.com/CeresDB/ceresdbproto/pull/71

Reply:
I re-thought the current implementation. Procedure should no longer hold a reference to Cluster, so there is no need to pass in ClusterName here. The information of the entire cluster is directly passed in through Request. This change is not included in this pull request.